be/src/common/config.h

```diff
@@ -1087,6 +1087,28 @@ CONF_Bool(datacache_tiered_cache_enable, "true");
 CONF_String(datacache_engine, "");
 // The interval time (millisecond) for agent report datacache metrics to FE.
 CONF_mInt32(report_datacache_metrics_interval_ms, "60000");
+// Whether enable automatically adjust cache space quota.
+// If true, the cache will choose an appropriate quota based on the current remaining space as the quota.
+// and the quota also will be changed dynamiclly.
+// Once the disk space usage reach the urgent level, the quota will be decreased to keep the disk usage
+// around the disk safe level.
+// On the other hand, if the cache is full and the disk usage falls below the disk safe level for a long time,
+// which is configured by `datacache_disk_idle_period_for_expansion`, the cache quota will be increased to keep the
+// disk usage around the disk safe level.
+CONF_mBool(datacache_auto_adjust_enable, "false");
+// The disk usage threshold, which trigger the cache eviction and quota decrased.
+CONF_mInt64(datacache_disk_urgent_level, "80");
+// The disk usage threshold, the cache quota will be decrease to this level once it reach the urgent level.
+CONF_mInt64(datacache_disk_safe_level, "70");
```

Comment:
```suggestion
// The disk usage threshold, the cache quota will be decreased to this level once it reach the urgent level.
CONF_mInt64(datacache_disk_safe_level, "70");
```

didn't get the point, once the disk usage reaches 80%, the disk cache quota will be set to 70% immediately, is it?

Reply:
> didn't get the point, once the disk usage reaches 80%, the disk cache quota will be set to 70% immediately, is it?

Yes, you are right.